docs/content/install.md

```diff
@@ -58,29 +58,31 @@ If you use a cluster domain other than `cluster.local` set it by using the `clus
 helm install maesh maesh/maesh --set clusterDomain=my.custom.domain.com
 ```
 
-## Service Mesh Interface
+## Access Control List
```

Comment:
Should we maybe add, that it was used to be called "SMI" before?

Reply:
I don't know, It's not really the same thing. How would you write it?